source/proxysql-conf.rst

```diff
@@ -394,7 +394,7 @@ which will have the following output::
 
 The next command will print you the needed admin password::
 
-  $ kubectl get secrets $(kubectl get pxc -o jsonpath='{.items[].spec.secretsName}') -o template='{{ .data.proxyadmin | base64decode }}'
+  $ kubectl get secrets $(kubectl get pxc -o jsonpath='{.items[].spec.secretsName}') -o template='{{ .data.proxyadmin | base64 --decode }}'
```

Comment:
We cannot change `base64decode` to `base64 --decode` in a template as it won't be recognized.